webdriver-spec.html

```diff
@@ -1435,288 +1435,513 @@
+to describe the full feature set for a session.
+
+<p>The following table provides an overview of the standard capabilities that
+each implementation <em class="rfc2119" title="MUST">MUST</em> support.
+An implementation <em class="rfc2119" title="MAY">MAY</em> define additional
+capabilities, although it is
+<em class="rfc2119" title="RECOMMENDED">RECOMMENDED</em> that custom
+capabilities prepend the capability key with a vendor prefix, as outlined in
+[[CSS21]].
+
+<aside class="example">
+<p>As an example, Mozilla could elect to hide new features behind capabilities
+with a "-moz-" prefix:
+
+<pre class=highlight>{
+  "browserName": "firefox",
+  "browserVersion": "1234",
+  "-moz-experimental-webdriver": true
+}</pre>
+</aside>
 
 <table class=simple>
- <tr>
+<thead>
+  <tr>
   <th>Capability
   <th>Key
   <th>Value Type
   <th>Description
- </tr>
+  </tr>
+</thead>
 
- <tr>
+<tbody>
+  <tr>
   <td><dfn>Browser name</dfn>
   <td>"<code>browserName</code>"
   <td>string
   <td>Identifies the user agent.
- </tr>
+  </tr>
 
- <tr>
+  <tr>
   <td><dfn>Browser version</dfn>
   <td>"<code>browserVersion</code>"
   <td>string
   <td>Identifies the version of the user agent.
- </tr>
+  </tr>
 
- <tr>
+  <tr>
   <td><dfn>Platform name</dfn>
   <td>"<code>platformName</code>"
   <td>string
   <td>Identifies the system or operating system
-   of the <a>endpoint node</a>.
- </tr>
+  of the <a>endpoint node</a>.
+  </tr>
 
- <tr>
+  <tr>
   <td><dfn>Platform version</dfn>
   <td>"<code>platformVersion</code>"
   <td>string
   <td>Identifies the system version of the <a>endpoint node</a>.
- </tr>
+  </tr>
 
- <tr>
+  <tr>
   <td><dfn>Accept SSL certificates</dfn>
   <td>"<code>acceptSslCerts</code>"
   <td>boolean
-  <td>Indicates that untrusted and self-signed SSL certificates
-   are trusted implicitly on <a data-lt=go>navigation</a>
-   for the duration of the session.
- </tr>
-</table>
-
-<p>Implementations are free to use the <a>capabilities</a> dictionary
- passed in the request to create a <a>new session</a>
- to provide additional feature requests
- from any of the <a>remote ends</a>, including the <a>endpoint node</a>.
- Likewise the <a>remote end</a> may supply additional entries
- on the <a>capabilities</a> object returned
- from creating a <a>new session</a>.
+  <td>Indicates whether untrusted and self-signed SSL certificates are
+  implicitly trusted on <a data-lt=go>navigation</a> for the duration of the
+  session.
+  </tr>
 
-<aside class=example>
- <p>Capabilities may for example be used
-  to provide a serialised profile for a <a>remote end</a> to use
-  during startup of the user agent binary.
+  <tr>
+  <td><dfn>Page Load Strategy</dfn>
+  <td>"<code>pageLoadStrategy</code>"
+  <td>string
+  <td>Defines the <a>current session</a>’s <a>page load strategy</a>.
+  </tr>
 
- <p>A fictive example of this:
+  <tr>
+  <td><dfn>Proxy Configuration</dfn>
+  <td>"<code>proxy</code>"
```

Comment:
We should probably thoroughly describe the proxy object, but that could wait for a follow edit.